spec/integration/support/command_line_examples.rb

```diff
@@ -68,6 +68,26 @@
           )
         ).to fail.and include_stderr("The following scope is not supported: foobar")
       end
+
+      it "does not show a hint that the inspection result can be shown by using the show " \
```

Comment:
I would reduce the size of the description, for example `does not print show hint if --show is used`